tests/integration/testmarketplace.nim

```diff
@@ -144,16 +225,20 @@ marketplacesuite "Marketplace payouts":
     NodeConfigs(
       # Uncomment to start Hardhat automatically, typically so logs can be inspected locally
       hardhat: HardhatConfig.none,
-      clients: CodexConfigs.init(nodes = 1)
-      # .debug() # uncomment to enable console log output.debug()
-      # .withLogFile() # uncomment to output log file to tests/integration/logs/<start_datetime> <suite_name>/<test_name>/<node_role>_<node_idx>.log
-      # .withLogTopics("node", "erasure")
-      .some,
-      providers: CodexConfigs.init(nodes = 1)
-      # .debug() # uncomment to enable console log output
-      # .withLogFile() # uncomment to output log file to tests/integration/logs/<start_datetime> <suite_name>/<test_name>/<node_role>_<node_idx>.log
-      # .withLogTopics("node", "marketplace", "sales", "reservations", "node", "proving", "clock")
-      .some,
+      clients: CodexConfigs
+        .init(nodes = 1)
+        #  .debug() # uncomment to enable console log output.debug()
+        .withLogFile()
+        # uncomment to output log file to tests/integration/logs/<start_datetime> <suite_name>/<test_name>/<node_role>_<node_idx>.log
+        .withLogTopics("node", "erasure").some,
+      providers: CodexConfigs
+        .init(nodes = 1)
+        #  .debug() # uncomment to enable console log output
+        .withLogFile()
```

Comment:
Probably better to avoid committing with `withLogFile` enabled.
```suggestion
        # uncomment to output log file to tests/integration/logs/<start_datetime> <suite_name>/<test_name>/<node_role>_<node_idx>.log
        .withLogTopics("node", "erasure").some,
      providers: CodexConfigs
        .init(nodes = 1)
        #  .debug() # uncomment to enable console log output
```